frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -16,91 +16,60 @@ export interface EmptySProps {
-                to={`/deploy-image?preselected-ns=${activeNamespace}`}
-              >
-                Deploy Image
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Browse Catalog</CardHeader>
-            <CardBody>Browse the catalog to discover, deploy and connect to services</CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/catalog">
-                Browse Catalog
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Import from Dockerfile</CardHeader>
-            <CardBody>Import your Dockerfile from your git repo to be built & deployed</CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/import?importType=docker">
-                Import from Dockerfile
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Import YAML</CardHeader>
-            <CardBody>Create or replace resources from their YAML or JSON definitions.</CardBody>
-            <CardFooter>
-              <Link
-                className="pf-c-button pf-m-secondary"
-                to={formatNamespacedRouteForResource('import', activeNamespace)}
-              >
-                Import YAML
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Add Database</CardHeader>
-            <CardBody>
-              Browse the catalog to discover database services to add to your application
-            </CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/catalog?category=databases">
-                Add Database
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-      </Grid>
-    </div>
-  </React.Fragment>
-);
+const ODCEmptyState: React.FC<Props> = ({ title, activeNamespace }) => {
+  const navigateTo = (url: string) => {
+    history.push(url);
+  };
+
+  return (
+    <React.Fragment>
+      <div className="odc-empty-state__title">
+        <PageHeading title={title} />
+        <p className="co-catalog-page__description">
+          Select a way to create an application, component or service from one of the options.
+        </p>
+      </div>
+      <div className="odc-empty-state__content">
+        <CatalogTile
+          onClick={() => navigateTo('/import?importType=git')}
+          title="From Git"
+          iconImg={getImportForIconClass('from-git-icon')}
```

Comment:
You can use font awesome icons for most of the cases here. Instead of sending `iconImg` prop you can send `iconClass` with fa classes.

Reply:
@rohitkrai03 Not found the same icon for all options. But used class for those options whose icon is available.

Reply:
Can you use PF4 icons instead? They have every icon except the docker file icon that I could find. Is UX dead set on this new docker icon? cc @serenamarie125

Reply:
PF4 has this icon
![image](https://user-images.githubusercontent.com/14068621/63469301-df4b2500-c437-11e9-9866-08b7873731b7.png)

Reply:
@christianvogt Pf4 give component for the icon but `catalogTile` component is not supporting component for the icon. Its supports icon class or icon path.

Reply:
@christianvogt we aren't allowed to use that Docker icon, so we created a new one. All the svgs for the icons are located in the folder below: 
https://drive.google.com/drive/u/0/folders/1CxAQhgl14ySY5bkM2A_v6nyZUcBLbk4q